Exercise5/Main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <iomanip>
+
+
+int main()
+{
+	int myFirstArray[10];
+	int number = -1;
+
+	std::cout << "Enter a number if you want: ";
```

Comment:
You could place these two lines at the beginning of the for loop. That way you don't need to repeat them in lines 32 & 33